grants/templates/grants/application_review.html

```diff
@@ -0,0 +1,83 @@
+{% extends "wafer/base.html" %}
```

Comment:
We'll need to talk about the review process later on still so I wont look at this yet.